schema/sale_artwork.js

```diff
@@ -40,17 +35,6 @@ const SaleArtworkType = new GraphQLObjectType({
       bidder_positions_count: {
         type: GraphQLInt,
       },
-      bidder_positions: {
```

Comment:
I suppose it depends on whether or not we want to enforce the `me`-scoped queries. I was initially planning on using this on the artwork page auction interface but I'll try coming at it from `me` first.